### PR TITLE
Additional information for cards

### PR DESCRIPTION
**Pull request by Hunter2065**
# Checklist:

- [x] Added specs for all new ruby code
- [x] Green rspec run
- [x] Green rubocop
- [x] PR review from teammate

# What?
places' cards in list

## Why?
'cause it was a little bit inflammable. 

## How?
In cards I changed text on price when added that to it also added style for rating and added style for address and price changed on title 

## Testing?
some things

## Screenshots (optional)
image

## Anything Else?
no

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ ruby '3.0.0'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
 gem 'rails', '~> 6.1.3', '>= 6.1.3.2'
 # Use pg as the database for Active Record
-gem 'pg'
+gem 'pg', '~> 1.1'
```
Comment (VanyaNovak) [inline Gemfile:9]: Remove pg version

File: app/views/places/_place.html.slim [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+                  h5
+                    = place.title
+                  .col-12
+                    p New York, str Big Halls, 129a
+
+            .col-6
+              = link_to place
+                button.btn.btn-success type="button"
+                  = place.price
+                  | $
```
Comment (KostyaKobel) [inline app/views/places/_place.html.slim:35]: Add new empty line

File: Gemfile (hunk 1 above)
Comment (Hunter2065) [reply Gemfile:9]: fixed)

File: app/views/places/_place.html.slim (hunk 2 above)
Comment (Hunter2065) [reply app/views/places/_place.html.slim:35]: okay)

Comment (Hunter2065) [reply app/views/places/_place.html.slim:35]: fixed

Comment (KostyaKobel) [reply app/views/places/_place.html.slim:36]: Removed empty line on 21 string and add empty line after 35 string

Comment (Hunter2065) [reply app/views/places/_place.html.slim:35]: everywhere I have empty line in the end of files)

**Review by VanyaNovak — APPROVED** [review]
LGTM!

**Review by KostyaKobel — APPROVED** [review]
Fine

File: app/frontend/stylesheets/places.scss [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 html, body {
     // overflow-x: hidden;
-    background: #262F30 
+    background: #262F30
```
Comment (andrjuha01) [inline app/frontend/stylesheets/places.scss:7]: add semicolomn

File: db/seeds.rb [hunk 4]
```diff
@@ -7,16 +7,18 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 20.times do |n|
-  title  = Faker::Company.name
+  title  = Faker::Address.street_name
```
Comment (andrjuha01) [inline db/seeds.rb:10]: leave company

File: spec/controllers/places_controller_spec.rb [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe PlacesController do
   describe PlacesController do
-    it 'test rendering template' do
+    it 'render template index and partials' do
```
Comment (andrjuha01) [inline spec/controllers/places_controller_spec.rb:5]: it 'renders index template'


partials are just part of template

File: app/frontend/stylesheets/places.scss (hunk 3 above)
Comment (Hunter2065) [reply app/frontend/stylesheets/places.scss:7]: fixed)

File: db/seeds.rb (hunk 4 above)
Comment (Hunter2065) [reply db/seeds.rb:10]: Okay), fixed)

File: spec/controllers/places_controller_spec.rb (hunk 5 above)
Comment (Hunter2065) [reply spec/controllers/places_controller_spec.rb]: yap, I forget delete it after trying to test rendering of partials, fixed)

**Pull request merged**